draft-ietf-anima-brski-prm.md

```diff
@@ -336,84 +320,84 @@ Solution examples based on existing technology are provided with the focus on ex
+    |                    | S igning      | Tracker   |
+    |                    | A uthority    |           |
+    |                    +---------------+-----------+
+    |                                         ^
+    |                                         | BRSKI-
+    |                                         | MASA
+    |          ...............................|.........
+    V          .                              v        .
++--------+     .  +------------+        +-----------+  .
+|        |     .  |            |        |           |  .
+| Pledge | BRSKI- | Registrar- | BRSKI- | Domain    |  .
+|        |  PRM   | Agent      |  EST   | Registrar |  .
+|        |<------>|            |<------>| (PKI RA)  |  .
+|        |     .  |     LDevID |        |           |  .
+|        |     .  +------------+        +-----+-----+  .
+| IDevID |     .                              |        .
+|        |     .           +------------------+-----+  .
++--------+     .           | Key Infrastructure     |  .
+               .           | (e.g., PKI Certificate |  .
+               .           |        Authority)      |  .
+               .           +------------------------+  .
+               .........................................
+                         "Domain" Components
 ~~~~
-{: #uc2figure title='BRSKI-PRM Architecture overview using registrar-agent' artwork-align="left"}
+{: #uc2figure title='BRSKI-PRM architecture overview using registrar-agent' artwork-align="left"}
 
 The following list describes the components in a (customer) site domain:
 
 * Pledge: The pledge is expected to respond with the necessary data objects for bootstrapping to the registrar-agent.
   The protocol used between the pledge and the registrar-agent is assumed to be HTTP in the context of this document.
-  Other protocols may be used like CoAP, Bluetooth, or NFC, but are out of scope of this document.
-  A pledge acting as a server during bootstrapping leads to some differences to BRSKI:
+  Other protocols such as CoAP, NFC, or Bluetooth may be used, but are out of scope of this document.
+  A pledge acting as a server during bootstrapping leads to some differences for BRSKI:
 
   * Discovery of the pledge by the registrar-agent must be possible.
 
   * As the registrar-agent must be able to request data objects for bootstrapping of the pledge, the pledge must offer corresponding endpoints.
 
-  * The registrar-agent MAY provide additional data to the pledge in the context of the voucher triggering request, that the pledge includes into the PVR.
+  * The registrar-agent MAY provide additional data to the pledge in the context of the voucher-request trigger, which the pledge includes into the PVR.
     This allows the registrar to identify, with which registrar-agent the pledge was in contact.
 
   * Order of exchanges in the call flow may be different as the registrar-agent collects both, PVR and PER, at once and provides them to the registrar.
```

Comment:
What exchanges? Different from what? There was an implicit mentioning of two exchanges: voucher-request trigger and enrollment-request trigger, which return PVR and PER, resp. How can they be collected "at once" when there are two separate HTTP requests? Do you just mean these two request can be in any order?

Probably it is the wrong place to talk about flows anyway, as this is only discussed in Section 6.

Reply:
The intention was to distinguish the order from BRSKI. Will change it to 

_Order of exchanges in the BRSKI-PRM call flow is different that in BRSKI, as the PVR and PER are collected at once and provided to the registrar._